### PR TITLE
[gazebo_plugins] address warnings

### PR DESCRIPTION
**Pull request by Karsten1987**
pretty trivial change to avoid another warning.

### CODE REVIEW

File: gazebo_plugins/test/test_gazebo_ros_diff_drive.cpp [hunk 1]
```diff
@@ -126,7 +126,7 @@ INSTANTIATE_TEST_CASE_P(
   GazeboRosDiffDrive, GazeboRosDiffDriveTest, ::testing::Values(
     TestParams({"worlds/gazebo_ros_diff_drive.world"}),
     TestParams({"worlds/gazebo_ros_skid_steer_drive.world"})
-));
+),);
```
Comment (jacobperron) [inline gazebo_plugins/test/test_gazebo_ros_diff_drive.cpp]: Looks like the linters don't like this change.

Comment (Karsten1987) [reply gazebo_plugins/test/test_gazebo_ros_diff_drive.cpp]: should be addressed in 90c45fc

**Review by jacobperron — APPROVED** [review]
LGTM with green CI

**Comment by Karsten1987** [comment]
I'll leave it up to @chapulina or @ahcorde to run CI. Not sure which build job is run for gazebo related builds.

**Comment by scpeters** [comment]
it looks like there are just some flaky test failures

**Comment by jacobperron** [comment]
@ros-pull-request-builder retest this please

**Comment by jacobperron** [comment]
Yeah, there seems to be number of flakes.

**Comment by Karsten1987** [comment]
I believe these changes are unrelated to flakey tests given that this PR only tackles compile time things.

**Comment by jacobperron** [comment]
Linux: [Build Status](https://ci.ros2.org/job/ci_linux/12064/)

**Comment by scpeters** [comment]
yes, remaining failures look flaky

**Review by scpeters — APPROVED** [review]

**Pull request merged**